catalog/dags/database/batched_update/batched_update.py

```diff
@@ -114,7 +121,7 @@ def update_batches(
             sql_template=constants.UPDATE_BATCH_QUERY,
             query_id=query_id,
             # Only log the query the first time, so as not to spam the logs
-            log_sql=batch_start == 1,
+            log_sql=batch_start == initial_batch_start,
```

Comment:
It looks like this was `1` before but with the new default it will be `0`. This seems like the correct value now (and maybe was the wrong value before?) but I just wanted to double check!

Reply:
Correct! I had a very small commit ready to fix it to 0, which ended up being superseded by this PR.